content/20.pharmaceuticals.md

```diff
@@ -651,6 +653,26 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+##### Anakinra
+
+Anakinra is a recombinant of the IL-1 receptor antagonist (IL-1Ra) and blocks the binding of IL-1α and IL-1β to IL-1R1.
+Its primary function is to reduce inflammation by mediating IL-1, providing a broad spectrum of applications that include the treatment of acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
+Anakinra has a favorable safety profile with very low rates of reported side effects in therapeutic studies for rheumatoid arthritis [@doi:10.1136/ard.2005.048371; @doi:10.1093/rheumatology/kew208; @url:https://www.globenewswire.com/news-release/2020/12/22/2149573/0/en/FDA-Approves-KINERET-anakinra-for-the-Treatment-of-Deficiency-of-IL-1-Receptor-Antagonist-DIRA.html].
+
+To date, a couple of cohort studies have been conducted to determine the therapeutic value of anakinra. 
+The first study at the San Raffaele Hospital in Milan, Italy, intravenously treated 29 patients with high doses of anakinra versus 16 controls, all of whom received the standard of care at the time [@doi:10.1016/S2665-9913(20)30127-2]. 
+A second cohort study at the Groupe Hospitalier Paris Saint-Joseph in Paris, France, subcutaneously administered anakinra to 52 patients who also received standard care at the time versus 44 historical control subjects. 
+In both studies, anakinra seemed to improve respiratory function, decrease the likelihood of requiring invasive mechanical ventilation, and in Paris, it significantly reduced mortality [@doi:10.1016/S2665-9913(20)30164-8]. 
+Similar observation was made in [@doi:10.1007/s11739-020-02600-z], where anakinra was used as a second tier after no response to standard corticosteroids or combination of corticosteroids and tocilizumab treatments.
+
+In another retrospective cohort study carried by Radboud University Medical Center (Nijmegen, The Netherlands) and Bernhoven Hospital (Uden, The Netherlands), Anakinra was found to effectively reduce hyperinflammation clinical signs for critically ill patients.[@doi:10.1186/s13054-020-03364-w]. 
```

Comment:
```suggestion
In another retrospective cohort study carried by Radboud University Medical Center (Nijmegen, The Netherlands) and Bernhoven Hospital (Uden, The Netherlands), Anakinra was found to effectively reduce hyperinflammation clinical signs for critically ill patients [@doi:10.1186/s13054-020-03364-w]. 
```

As above, I recommend being more quantitative about what we mean by "effectively reduce".